test/broadcast.jl

```diff
@@ -409,3 +409,9 @@ Base.Broadcast.broadcast_c(f, ::Type{Array19745}, A, Bs...) =
     @test isa(aa .+ 1, Array19745)
     @test isa(aa .* aa', Array19745)
 end
+
+# broadcast should only "peel" one container level
+let io = IOBuffer()
+    broadcast(x -> print(io, x), [Nullable(1.0)])
+    @test String(take!(io)) == "Nullable{Float64}(1.0)"
```

Comment:
Sounds a bit weird to use string comparison to test the type of the result. Why not e.g. call `push!` to add values to a vector instead?